apis/axios.ts

```diff
@@ -24,6 +26,13 @@ export const instance = axios.create({
   },
 });
 
+export const imageRequestInstance = axios.create({
+  baseURL: process.env.NEXT_PUBLIC_BASE_API_URL,
+  headers: {
+    "Content-Type": "multipart/form-data",
+  },
+});
+
 instance.interceptors.request.use(async (config) => {
   try {
     if (getIsServer()) {
```

Comment:
p3: 
```suggestion
    if (getIsServer()&&context) {
```
로 수정해주시고 아래 옵셔널 지워주시면 더 좋을거같아요